guardian/gateway
# Clean up image / no image a/b tests

## What does this change?



Removes a/b tests introduced in guardian/gateway/pull/1569 and guardian/gateway/pull/1584, with the behaviour now permanently changed to be the same as the variants with no image.

## How to test



Could check out the code locally, and verify that the image is no longer shown on `/consents/communications` and `/signin/success`. May be sufficient to just note that the images no longer show in Chromatic, though. 

## Images

Can see previous PRs linked above for images.

## Accessibility
